README.md

```diff
@@ -2,3 +2,5 @@ LMS1xx
 ======
 
 ROS driver for the SICK LMS1xx family of laser scanners. Originally from [RCPRG](https://github.com/RCPRG-ros-pkg/RCPRG_laser_drivers).
+
+Added the ability for the user to specify the desired start and stop angles in the ROS frame. 
```

Comment:
Please revert this README change— the changelog is the place for information about development history.